tuftool/Cargo.toml

```diff
@@ -31,7 +32,11 @@ ring = { version = "0.17", features = ["std"] }
 serde = "1"
 serde_json = "1"
 simplelog = "0.12"
+sigstore = { git = "https://github.com/fghanmi/sigstore-rs.git", branch = "target_v1" }
```

Comment:
Is it ok for this to point at a personal fork?

Reply:
This was intentional, it was agreed with @bkabrda 
because https://github.com/sigstore/sigstore-rs/pull/385 is still not merged yet (and tough not at it's latest version)